version: '3.7'
services:
  delugevpnuser1:
    image: binhex/arch-delugevpn:latest
    container_name: delugevpnuser1
    restart: unless-stopped
    networks:
      - ${NETWORK}
    cap_add:
      - NET_ADMIN
    environment:
      - PUID=${PUID}
      - PGID=${PGID}
      - VPN_ENABLED=yes
      - VPN_USER=${VPN_USER}
      - VPN_PASS=${VPN_PASS}
      - VPN_PROV=${VPN_PROV}
      - STRICT_PORT_FORWARD=yes
      - ENABLE_PRIVOXY=yes
      - LAN_NETWORK=${LAN_NETWORK}
      - NAME_SERVERS=209.222.18.222,37.235.1.174,8.8.8.8,209.222.18.218,37.235.1.177,8.8.4.4
      - UMASK=000
      - DEBUG=false
    volumes:
      - /delugevpnuser1:/config
      - /etc/localtime:/etc/localtime:ro
      - /seedbox:/data
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.delugevpnuser1.rule=Host(`${USER1}.${TRAEFIK_DOMAIN}`)"
      - "traefik.http.routers.delugevpnuser1.middlewares=common-auth@file"
      
volumes:
  delugevpnuser1:
    driver: local-persist
    driver_opts:
      mountpoint: /data/config/delugevpnuser1
  seedbox:
    driver: local-persist
    driver_opts:
      mountpoint: /data/config/nextcloud-data/${USER1}/files/seedbox
